common/lib/xmodule/xmodule/course_module.py

```diff
@@ -830,6 +830,13 @@ class CourseFields(object):
         },
         scope=Scope.settings
     )
+    allow_unsupported_xblocks = Boolean(
+        display_name=_("Allow the creation of unsupported XBlocks"),
```

Comment:
Yes, note that this feature is not specific to problem types-- it controls HTML templates and XBlocks as well (really any xmodule/xblock that can be added through the "New" buttons).